.github/workflows/run-benchmark.yml

```diff
@@ -0,0 +1,163 @@
+name: benchmark
+on:
+  workflow_call:
+
+jobs:
+  benchmark-vs-thresholds:
+    # https://runs-on.com/features/custom-runners/
+    runs-on:
+      labels:
+        - runs-on
+        - runner=2cpu-4ram
+        - run-id=${{ github.run_id }}
+
+    container: swift:noble
+
+    defaults:
+      run:
+        shell: bash --noprofile --norc -e -u -o pipefail {0}
+
+    env:
+      PR_COMMENT: null # will be populated later
```

Comment:
I don't think you need this?

Reply:
the vscode extension complain, otherwise no.

Reply:
removed the env var so i guess we're good.